Makefile

```diff
@@ -175,26 +182,21 @@ help:
 	@echo
 
 
+
+
 # TODO: add targets `translate` when merged
 .PHONY: all
 all: setup config-templates lint build
 
 
 .PHONY: setup
-setup: $(VENV) $(NODE_MODULES)
-
+setup: $(NODE_MODULES)
+	if ! [ -d "$(VENV)" ] ;then pipenv install ;fi
```

Comment:
dev dependencies are missing `pipenv install --dev`

Reply:
For the serve to work it might be also needed to do as before the `pipenv install -e .`

Reply:
Seems that adding `pipenv install -e .` here fixed the issue when serving locally